salt/metalk8s/orchestrate/deploy_new_node.sls

```diff
@@ -52,22 +57,17 @@ Run the highstate:
   salt.state:
     - tgt: {{ pillar['node_name'] }}
     - highstate: True
+    - pillar: {{ pillar_data | tojson }}
     - require:
       - salt: Set grains
       - salt: Refresh the mine
 
 {%- if 'etcd' in pillar.get('metalk8s', {}).get('nodes', {}).get(pillar['node_name'], {}).get('roles', []) %}
 
-  {%- set node_ip = grains['metalk8s']['control_plane_ip'] %}
-
-  {#- etcd endpoint of the new node. #}
-  {%- set endpoint = 'https://' ~ node_ip ~ ':2380' %}
-
 Register the node into etcd cluster:
   module.run:
     - metalk8s.add_etcd_node:
       - host: {{ pillar['node_name'] }}
-      - endpoint: {{ endpoint }}
```

Comment:
That's cheating :stuck_out_tongue: But OK :laughing:

Reply:
It's just a hack waiting for #877 :stuck_out_tongue: